test/user.test.js

```diff
@@ -0,0 +1,100 @@
+const app = require('../src/index.js');
+const chai = require("chai");
+const chaiHttp = require("chai-http");
+const { expect } = chai;
+const { before, describe, it } = require('mocha');
+chai.use(chaiHttp);
+
+describe('user', () => {
+    let token, uid;
+
+    before('create user and get user api token', function(done) {
```

Comment:
These tests should probably be replicated for the administrators (I think you might find a bug as mentioned previously).